scripts/platform_agnostic_towncrier.py

```diff
@@ -0,0 +1,13 @@
+from subprocess import check_call
+
+
+def main():
+    """
+    Platform agnostic towncrier to support tox -e docs on a windows based platform natively.
+    Previously we used 'sh -c' which will not work on pycharm terminal in windows for example.
+    """
+    check_call("towncrier --draft > doc/en/_changelog_towncrier_draft.rst", shell=True)
```

Comment:
shell=True is generally a security problem -- you can do this in pure python with:

```python
with open(..., 'w') as f:
    subprocess.check_call(('towncrier', '--draft'), stdout=f)
```